### ISSUE TITLE
`unscoped native element` warning

### ISSUE THREAD

**Issue by hepiyellow** [wix/stylable#450]
I have this use case where I want to apply theme namespace:
 
> `theme.st.css`
```css
@namespace "BackofficeTheme";
```

> `someComp.st.css`
 ```
 :import {
   -st-from: "../theme.st.css";
   -st-default: BOTheme;
 }
 ```
 ![image](https://user-images.githubusercontent.com/18248989/44086713-17d87b26-9fc6-11e8-80e7-f674cce9854f.png)
 
 I am Getting
 `[stylable] unscoped native element "BOTheme" will affect all elements of the same type in the document`. 

Why is this?
It is not a native element.

**Comment by hepiyellow** [comment]
1. Maybe it should just be fixed to `unscoped custom element`.
1. By the way, what if I DO want to affect all elements of the same type in the document. Can I disable the warning?

**Comment by tomrav** [comment]
1. I agree the error should be more exact if you've used native/custom element, I'll be sure to fix it before our next release.
2. Short term, you should use the `@st-scope` feature once it's available (Soon. ™️). Later on, we'd like to expand our diagnostics in general to allow more user flexibility.

**Comment by tomrav** [comment]
There's currently no way to turn off diagnostics in Stylable (we're hoping to get to it at some point).

Regardless of that, how are these warnings from build getting to the browser? Are you running webpack-dev-server?

As an aside, I hope to release the new `@st-scope` feature today, so you could wait and use that instead.

**Comment by tomrav** [comment]
Resolved in #455 
The error message has been made more clear, and `@st-scope` feature introduced to allow for controlled scoping.
